docs/BreakingChanges/147 BinaryFormatter can fail to find type from LoadFrom context.md

```diff
@@ -0,0 +1,44 @@
+## 147: BinaryFormatter can fail to find type from LoadFrom context
+
+### Scope
+Edge
+
+### Version Introduced
+4.5
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+As of .NET Framework 4.5, a number of XmlSerializer changes may cause differences in deserialization when using `BinaryFormatter` to deserialize types that had been loaded in the
```

Comment:
I see. Yea, that may work for an analyzer. It's too bad we can't look for the XmlSerializer, though, because I expect that the majority of BinaryFormatter usages don't also use an XmlSerializer (they're often treated as 'competing' options for how to serialize something). We may want to at least scope the analyzer to look for some XmlSerializer usage in the project, even if we don't know exactly what pattern to look for.